src/store.ts

```diff
@@ -17,6 +18,23 @@ export const isShowingPoints = writable(defaults.showPoints);
 export const initialViewport = writable(defaults.viewport);
 export const navMode = writable(NavMode.autofit);
 
+export function getFormSelections() {
+  try {
+    if (sessionStorage.getItem('form')) {
+      return JSON.parse(sessionStorage.getItem('form')!) as FormSelections;
+    }
+    return new FormSelections();
+  } catch {
+    sessionStorage.removeItem('form');
+    return new FormSelections();
+  }
```

Comment:
explanatory note and small simplification:

```suggestion
  try {
    if (sessionStorage.getItem('form')) {
      return JSON.parse(sessionStorage.getItem('form')!) as FormSelections;
    }
  } catch {
    // we are probably here because parsing failed, so remove bad JSON from sessionStorage
    sessionStorage.removeItem('form');
  }
  return new FormSelections();
```